java-lib/src/main/java/com/wavefront/ingester/AbstractIngesterFormatter.java

```diff
@@ -469,7 +469,8 @@ public static String unquote(String text) {
 
   @Nullable
   public static String getHost(@Nullable List<Annotation> annotations,
```

Comment:
lets rename it to `getHostAndNormalizeTabs`
and add ad `//TODO: review why we change tag to _tag here`